addons/controllers/antrea/antreaconfig_util.go

```diff
@@ -162,6 +165,10 @@ func (r *AntreaConfigReconciler) ClusterToAntreaConfig(o client.Object) []ctrl.R
 }
 
 func mapAntreaConfigSpec(cluster *clusterv1beta1.Cluster, config *cniv1alpha1.AntreaConfig) (*antreaConfigSpec, error) {
+
+	packageName := config.GetLabels()[constants.PackageNameLabel]
+	version := strings.TrimPrefix(strings.Split(packageName, "---")[0], "antrea.tanzu.vmware.com.")
```

Comment:
I think it's better to fetch version from Package spec.

Reply:
this lable is also used by other component, so i think make sense to reuse it for now.

Reply:
Do you mean other component also extract the Package version according to the label value? My suggestion is that it get the Package name from the label value, and do a get call to get the Package spec, and get the version from Package spec.

Reply:
I get your point, i just mean this label is used by other part of code too, so it's kind of reliable for a while. I would like to open a catchup pr later to refactor this part, if we don't have enough time for this fix.